containers/Comments/Comment.js

```diff
@@ -43,13 +48,22 @@ export const Comment = (props) => {
           />
         </Text>
       </View>
-      <Text style={styles.comment}>
-        {comment}
-      </Text>
-      <View style={styles.linkSection}>
-        <TouchableWithoutFeedback>
-          <Text style={styles.linkButton}>Weiterlesen</Text>
+      <View>
+        {!showWholeComment &&
+          <Text style={styles.comment} numberOfLines={2}>
+            {comment}
+          </Text>
+        }
+        {showWholeComment &&
+        <Text style={styles.comment}>
+          {comment}
+        </Text>
+        }
+        <TouchableWithoutFeedback onPress={toggleWholeComment}>
```

Comment:
we could hide `read more/less` if chars doesn't exceed maximum (for later). Adding an issue/improvement